optics-extra/src/Control/Parallel/Strategies/Optics.hs

```diff
@@ -0,0 +1,42 @@
+module Control.Parallel.Strategies.Optics
+  ( parOver
+  , evalOf
+  , parOf
+  ) where
+
+import qualified Control.Parallel.Strategies as Parallel
+import Optics.Core
+
+-- | Apply a traversal as a modifier in parallel.
+parOver :: Is k A_Traversal => Parallel.Strategy b -> Optic k is s t a b -> (a -> b) -> s -> t
+parOver strat o f = Parallel.runEval . traverseOf o (Parallel.rparWith strat . f)
+{-# INLINE parOver #-}
+
+-- | Evaluate the targets of a 'Lens' or 'Traversal' into a data structure
+-- according to the given 'Strategy'.
+--
+-- @
+-- 'evalTraversable' = 'evalOf' 'traverse' = 'traverse'
+-- 'evalOf' = 'id'
+-- @
+--
+-- @
+-- 'evalOf' :: 'Lens'' s a -> 'Strategy' a -> 'Strategy' s
```

Comment:
We don't need to add these as `lens` does since our type signatures are clearer.